packages/asset-swapper/src/utils/market_operation_utils/path_optimizer.ts

```diff
@@ -319,6 +307,13 @@ export function findOptimalRustPathFromSamples(
         fees,
         neonRouterNumSamples,
     );
+    // tslint:disable-next-line: no-unused-expression
+    samplerMetrics &&
+        samplerMetrics.logRouterDetails({
+            router: 'neon-router',
+            type: 'all',
+            timingMs: performance.now() - beforeTimeMs,
```

Comment:
NIT: you could use `beforeAllMs` here, then you could create `beforeTimeMs` inside the vip block and wouldn't need to reassign it. Same results but might be slightly more readable? 🤔